### DISCUSSION TITLE
DoubleML for Targetted Maximum Likelihood Estimation

### DISCUSSION THREAD

**Discussion by vivkhatri22** [DoubleML/doubleml-for-r#172] [General]
Good day....Apologies if this question is too large, but I am trying to understand the pros an cons of each approach for using ML for causal inference. Please advise. Thank you in advance........... Vivek

**Comment by PhilippBach** [comment]
Dear @vivkhatri22 ,

thanks for your interest in our paper and the Double Machine Learning approach, in more general. 

Indeed the question is very general so it is very difficult to give an appropriate answer. However, we get this question relatively frequently (what is better DoubleML or approach X?)?

An advantage of DoubleML that we really appreciate is in its generality. Hence, estimation of causal parameters as of [Chernozhukov et al. (2018)](https://arxiv.org/abs/1608.00060) is possible in a great number of causal models. The key requirement then is the existence of a Neyman orthogonal score. Some examples include

* Mediation analysis (see [Huber et al. (2021)](https://arxiv.org/abs/2002.12710))
* Sample selection models (see [Huber et al. (2022)](https://arxiv.org/abs/2012.00745))
* Gaussian graphical models (see [Klaassen et al. (2022)](https://arxiv.org/abs/1808.10532))
* Transformation models (see [Klaassen et al. (2022)](https://arxiv.org/abs/1712.07364))
* Estimation of Conditional Average Treatment Effects (CATEs) (see [Semenova and Chernozhukov (2020)](https://arxiv.org/abs/1702.06240))

.... just to name a few examples

However it's really hard to claim that one estimation approach (like for example DoubleML) is superior to others. I think, in the end this is also an empirical question, but I'm not aware of any extensive simulation or empirical comparison, for example to targeted maximum likelihood or so

If you or others are aware of such a comparison or want to add any comments, please feel free to add your reply here!

Best,

Philipp
